examples/ex02_cksum/test/ChecksumRTL_test.py

```diff
@@ -115,19 +116,16 @@ def setup_method( s, method ):
   # [teardown_method] will be called by pytest after executing each class method.
   # See pytest documetnation for more details.
   def teardown_method( s, method ):
-      s.vcd_file_name = ""
+    s.vcd_file_name = ""
 
   def run_sim( s, th, max_cycles=1000 ):
 
     # Check for vcd dumping
     if s.vcd_file_name:
-      th.dump_vcd = True
-      th.vcd_file_name = s.vcd_file_name
+      th.config_tracing = TracingConfigs(tracing='text_fancy', vcd_file_name=s.vcd_file_name)
```

Comment:
If `TracingConfigs` are options for tracing in general, then this option probably should have names like `output_name` or something?

Reply:
`vcd` makes it too specific to a single option

Reply:
only vcd pass will look at vcd_file_name, and other tracing passes probably have their own file name option because there can be cases where you want to dump textwave to one file and vcd to another file.